projects/js-packages/social-logos/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "social-logos",
+	"name": "@automattic/social-logos",
```

Comment:
Changing this will mean abandoning the listing at https://www.npmjs.com/package/social-logos in favor of a new one at https://www.npmjs.com/package/@automattic/social-logos. Do we really want to do that?

Reply:
I agree that this isn't a good idea. I've rolled that back here: c6d80a7